### ISSUE TITLE
DataLoom is no longer maintained

### ISSUE THREAD

**Issue by decaf-dev** [decaf-dev/obsidian-dataloom#958]
Hey guys. I have decided to no longer maintain DataLoom. With the release of the new Obsidian table editor in 1.5.0, DataLoom has become less relevant. I foresee that the Obsidian team will implement more features into this native editor that will be more suited for Obsidian ecosystem. 

DataLoom is also complex enough that I do not have time now to work on it by myself. From embedded views, to mobile support, there's a lot going on in this repository.

There are also key features missing from DataLoom that cannot be implemented, such as the ability to search through the files using the native Obsidian search pane.

I have a had a lot of fun developing DataLoom and I have appreciated all of the support.

Please open an issue if you have any issues exporting your data.

I also recommend that you check out the [Vault Explorer plugin](https://github.com/decaf-dev/obsidian-vault-explorer). It mimics the "sources" feature of DataLoom. It has been built in svelte for easier contributions, smaller build size, and faster speed.

Thanks!

**Comment by NewtonChutney** [comment]
Had a good run! Thanks a lot!

**Comment by timmolderez** [comment]
Many thanks for all the amazing work you've put into DataLoom! 🙏

**Comment by NewtonChutney** [comment]
Would you mark the repo archived.. in order to clarify the maintenance state?

**Comment by decaf-dev** [comment]
> Would you mark the repo archived.. in order to clarify the maintenance state?

I will in a few weeks if I don't get any issues for export bugs.

**Comment by NewtonChutney** [comment]
> > Would you mark the repo archived.. in order to clarify the maintenance state?
> 
> I will in a few weeks if I don't get any issues for export bugs.

Ahh, that's thoughtful of you, thanks for clarifying!

**Comment by DuckTapeKiller** [comment]
Will the plugin still be available for those who want to continue using it at their own risk? 

Or are you planning to withdraw it from Obsidian? 

I don't really want to move my tables to the native Obsidian tables.

**Comment by decaf-dev** [comment]
> Will the plugin still be available for those who want to continue using it at their own risk?
> 
> Or are you planning to withdraw it from Obsidian?
> 
> I don't really want to move my tables to the native Obsidian tables.

I can keep it as an available plugin, if that is helpful. I don't know what the Obsidian policy on this is though.

**Comment by DuckTapeKiller** [comment]
> I can keep it as an available plugin, if that is helpful. I don't know what the Obsidian policy on this is though.

I doubt they’ll remove it. I’ve found many plugins that haven’t been updated for two or three years, so I don’t see any reason why they would remove yours.

**Comment by kevinhar** [comment]
Thanks!!

**Comment by alkomy** [comment]
Thanks for developing such a good plugin, I used it in my vault, & it was one of main reasons to migrate from Notion.
Anyway, it's still working fine, so you can leave it in community plugins.
Respect

**Comment by BobWs** [comment]
Thanks for the plugin!

**Comment by BobWs** [comment]
All of a sudden, my plugin stopped working properly. When I open the Obsidian app on iOS (18.0.1), my Loom tables are blank. However, if I quit the app, remove it from the cache, and restart it, my Loom tables become visible again. But when I try to select another table from my list, I'm met with a blank page.

Has anyone else experienced this issue?

Is this problem related to iOS 18 or is it due to the plugin no longer being supported or actively developed?

Thanks in advance.

**Comment by decaf-dev** [comment]
@BobWs That's interesting. Could you take a screenshot for me and post it here? I have heard of iOS 18 having a few bugs.

Does the entire Obsidian app load property but only the loom tables are blank? If the Obsidian app is loading property and you can select files from the file explorer, there might have been a breaking change in the Obsidian API.

**Comment by BobWs** [comment]
> Does the entire Obsidian app load property but only the loom tables are blank? If the Obsidian app is loading property and you can select files from the file explorer, there might have been a breaking change in the Obsidian API.

Yes the Obsidian app loads properly, only the loom tables are blank. All other third part plug-ins work as far as I know.

see the screenshots, blank is when opening the app and select an table and screenshot 2 is after removing the app from the catch.

Even when I create a new table it is blank afterwards.

![IMG_0037](https://github.com/user-attachments/assets/cd19a255-ab37-4d64-8e2f-f0bc0bc72c1b)
![IMG_0038](https://github.com/user-attachments/assets/97dfe4cb-92e9-44fd-af01-119004c37b20)

**Comment by BobWs** [comment]
Turns out it's not only the ios app on my macbook the Obsidian desktop App also shows the same problem! (macOS 10.13.6) Obsidian Version 1.6.7 (Installer 1.4.16).

**Comment by decaf-dev** [comment]
Gotcha. It's definitely the plugin then. I have been thinking about picking
up this project again... I just don't have a lot of time.

**Comment by NewtonChutney** [comment]
> Is this issue out of date now? The readme has been updated

Ahh, apparently yes

**Comment by decaf-dev** [comment]
@ejektaflex I will not be continuing development on this project. Obsidian has updated their roadmap and they are working on dynamic views.
